generators/app/templates/test/index.js

```diff
@@ -18,7 +18,7 @@ describe("<%= description %>", () => {
 
     it(`should ${caseName.split("-").join(" ")}`, () => {
       const actual = babel.transformFileSync(actualFile, {
-        plugins: [plugin]
+        plugins: [plugin(babel)]
       }).code;
```

Comment:
Rather than specifying these here, there should be a `.babelrc` file in each test case folder next to `actual.js` and `expected.js`